mscore/pianoview.cpp

```diff
@@ -523,7 +523,11 @@ void PianoView::wheelEvent(QWheelEvent* event)
       else if (event->modifiers() == Qt::ControlModifier) {
             //Vertical zoom
             QRectF viewRect = mapToScene(viewport()->geometry()).boundingRect();
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 1)) //???
```

Comment:
No documentation found since when this replacement exists, it does not in 5.9, and docs are for 1.14.1